gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-core/src/main/resources/webroot/assets/css/login_flow.css

```diff
@@ -0,0 +1,630 @@
+.container {
+    --topBorderColor: #6A4FF7;
```

Comment:
A good practice is not define all the variables in a dedicated section and reuse them after : 

```
:root {
  --topBorderColor: #6A4FF7;
  --width: 222px;
  --backgroundColor: #6A4FF7;
  --textColor: #FFFFFF;
}
```

and use them in the css file.

That way the only part of the custom theme will be to generate a custom `:root { }` css element 

```

Reply:
done, :root {} added